src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -84,12 +87,18 @@
 
     private static final String CATALOG_OBJECT_MODEL_REGEXP = "^([^/]+/[^/]+)(/[^/][0-9]{12})?$";
 
+    private static final String SCRIPT_URL_REGEX = "^((http|https)://.*|\\$\\{PA\\_CATALOG\\_REST\\_URL\\})/buckets/([^/]+)/resources/([^/]+)(/revisions/([^/][0-9]{12}))?/raw";
```

Comment:
If you have an example of SCRIPT_URL_REGEX , could you put it above as a comment? (the same for the other regexp)

Reply:
Ok

Reply:
scripts can be defined by other protocols than http(s), e.g. file:// ftp://

Reply:
I think the pattern should only try to detect catalog urls

Reply:
In that case, can we accept such URL ? as it is done in the tests ? 
http://localhost:8080/catalog/buckets/cloud-automation-scripts/Service_Start/raw

Reply:
if I understood well you suggest to detect only catalog urls (${PA_CATALOG_REST_URL}) and remove http|https|file|ftp ?. If it is the case, selection script url will not pass in the tests. Should I remove these tests then ?
Thanks

Reply:
I don't quite understand, maybe you can explain me on monday.

Indeed, I suggest that we only try to parse catalog urls.